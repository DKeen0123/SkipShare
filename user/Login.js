import React, { Component } from 'react'
import { Text, View, StyleSheet } from 'react-native'

import CreateUser from './CreateUser';

export default class Login extends Component {
  render() {
    return (
      <View>
        <CreateUser />
      </View>
    )
  }
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    justifyContent: "center"
  }
});