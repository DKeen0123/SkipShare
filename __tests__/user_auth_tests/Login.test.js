import React from 'react';
import { shallow } from 'enzyme';
import Login from '../../components/user/Login';

const props = { type: 'Register' }
const login = shallow(<Login {...props} />);

it('renders without crashing', () => {
  expect(login).toMatchSnapshot();
});