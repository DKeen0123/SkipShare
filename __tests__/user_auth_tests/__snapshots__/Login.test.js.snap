// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<Component
  style={
    Object {
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Apollo(Apollo(CreateUser)) />
  <Button
    onPress={[Function]}
    title="Login"
  />
</Component>
`;
